pkgs/development/python-modules/breezy/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, buildPythonPackage, fetchPypi, configobj, six, fastimport, dulwich, launchpadlib, testtools }:
```

Comment:
Can you break this line? Ideally one variable per line to make diffs more readable.

Reply:
sure, fixed